packages/contracts/development/contracts/RealityETHReopenable_ERC20-4.0.sol

```diff
@@ -0,0 +1,114 @@
+// SPDX-License-Identifier: GPL-3.0-only
+
+pragma solidity ^0.8.20;
+
+import {IRealityETHReopenable_ERC20} from "./IRealityETHReopenable_ERC20.sol";
+import {RealityETH_ERC20_v4_0} from "./RealityETH_ERC20-4.0.sol";
+
+// solhint-disable-next-line contract-name-camelcase
+contract RealityETHReopenable_v4_0 is IRealityETHReopenable_ERC20, RealityETH_ERC20_v4_0 {
+    // Special value representing a question that was answered too soon.
```

Comment:
Maybe you could document the high level purpose of why you introduce a feature for UNresolved_Answer. And then its would be nice to know whether answered too soon is the only use-case, or whether one would imagine more of them. (Maybe this information could come up a tiny prologue comment before the contract declaration in this file)